rust-connectors/sources/mqtt/src/main.rs

```diff
@@ -20,6 +21,18 @@ struct MqttOpts {
     #[structopt(short, long)]
     mqtt_url: String,
 
+    #[structopt(long)]
+    mqtt_tls: bool,
+
+    #[structopt(long)]
+    mqtt_username: Option<String>,
+
+    #[structopt(long, env = "MQTT_PASSWORD", hide_env_values = true)]
```

Comment:
it's a nit but probably may need to hide from CLI options so people are disciplined to use ENV in dev without exposing password via CLI args.